src/com/palmergames/bukkit/towny/object/Town.java

```diff
@@ -64,6 +66,8 @@
 	private long registered;
 	private transient List<Invite> receivedinvites = new ArrayList<>();
 	private transient List<Invite> sentinvites = new ArrayList<>();
+	private HashSet<CustomDataField> metadata = null;
+	private boolean hasMeta;
```

Comment:
I am a little confused as to what the point of this boolean is? It seems more the information returned by this boolean can be conveyed by a simple check `metadata == null` method.

Reply:
Will do